apps/platform/src/app/(main)/(project)/[workspace]/[project]/@overview/page.tsx

```diff
@@ -0,0 +1,170 @@
+'use client'
+import { useEffect, useState } from 'react'
+import { KeyshadeLogoSVG, LockSVG } from '@public/svg/shared'
+import { useAtom, useAtomValue } from 'jotai'
+import { SecretSVG } from '@public/svg/dashboard'
+import OverviewLoader from '@/components/dashboard/overview/overviewLoader'
+import RegenerateKey from '@/components/dashboard/overview/RegenerateKey'
+import { projectPrivateKey, selectedProjectAtom } from '@/store'
+import { formatTimeAgo } from '@/lib/utils'
+import LocalKeySetup from '@/components/dashboard/overview/LocalKeySetup'
+import ServerKeySetup from '@/components/dashboard/overview/ServerKeySetup'
+import SetupLocalKeyDialog from '@/components/dashboard/overview/SetupLocalKeyDialog'
+
+function OverviewPage(): React.JSX.Element {
+  const [isLoading, setIsLoading] = useState(true)
+  const activeProject = useAtomValue(selectedProjectAtom)
+  const [privateKey, setPrivateKey] = useAtom(projectPrivateKey)
+  const [showAddKeyDialog, setShowAddKeyDialog] = useState(false)
+
+  useEffect(() => {
+    const key = activeProject?.storePrivateKey
+      ? activeProject.privateKey
+      : localStorage.getItem(`${activeProject?.name}_pk`) || null
+
+    setPrivateKey(key)
+    setIsLoading(false)
+  }, [activeProject, setPrivateKey])
+
+  const isKeyStoredOnServer = Boolean(activeProject?.privateKey)
+
+  if (isLoading || !activeProject) {
+    return (
+      <div className="space-y-4">
+        <OverviewLoader />
+        <OverviewLoader />
+        <OverviewLoader />
+      </div>
+    )
+  }
+
+  return (
+    <div className="flex h-full w-full gap-5 pt-5">
+      <div className="rounded-5 flex-1 bg-white/5 p-6 drop-shadow-[0px_1px_2px_rgba(16,24,40,0.06)]">
+        <div className="flex items-start justify-between pb-5">
+          <div className="flex flex-col gap-2">
+            <div className="flex items-center gap-3">
+              <KeyshadeLogoSVG />
+              <p className="text-2xl font-[700]">Keyshade</p>
+            </div>
+            <div className="text-base">{activeProject.description}</div>
+          </div>
+
+          <div className="rounded-[6px] bg-[#D9F7FF14] p-1.5 text-xs text-white">
+            {formatTimeAgo(new Date(activeProject.updatedAt))}
```

Comment:
i think we already have a function for this. `{dayjs(secret.updatedAt).toNow(true)} ago`